packages/contracts-bedrock/test/L1/SystemConfig.t.sol

```diff
@@ -362,7 +362,13 @@ contract SystemConfig_Init_CustomGasToken is SystemConfig_Init {
     )
         external
     {
+        // don't use vm's address
         vm.assume(_token != address(vm));
+        // don't use console's address
+        vm.assume(_token != address(0x000000000000000000636F6e736F6c652e6c6f67));
+        // don't use create2 deployer's address
+        vm.assume(_token != address(0x4e59b44847b379578588920cA78FbF26c0B4956C));
```

Comment:
Instead of hardcoding these, can we just use the existing variables from forge-std? https://github.com/foundry-rs/forge-std/blob/master/src/Base.sol